fiftyone/core/models.py

```
@@ -712,10 +801,14 @@ def _apply_video_model(
                 logger.warning("Sample: %s\nError: %s\n", sample.id, e)
 
 
-def _select_fields_for_inference(samples, model):
+def _select_fields_for_inference(samples, model, field_mapping=None):
```

Ruff (0.15.7)

[warning] Missing return type annotation for private function `_select_fields_for_inference`

(ANN202)